root_orchestrator/cloud-scheduler/cloud_scheduler.py

```diff
@@ -60,9 +58,9 @@ def replicate():
     job_id = data.get('job')
```

Comment:
I think this endpoint /api/calculate/replicate is deprecated. Can you remove it?